### PR TITLE
feat(uzf): new uzf output option - mean water content for all objects

### PR DESCRIPTION
**Pull request by emorway-usgs**
- Adds 'wc' keyword to UZF `OPTIONS` block
- New binary output file records mean water contents for every UZF object
- Includes new autotest that compares MODFLOW6/UZF6 output to MFNWT/UZF1 output recorded in the MT3D linker file (written by LMT package)
- Updates UZF definition file (.dfn) for supporting new binary output file with flopy
- Records these updates in the releaseNotes.tex file

### CODE REVIEW

File: doc/mf6io/mf6ivar/dfn/gwf-uzf.dfn [hunk 1]
```diff
@@ -51,6 +51,39 @@ optional true
...
+type record wc fileout wcfile
+shape
+reader urword
+tagged true
+optional true
+longname
+description
+
+block options
+name wc
```
Comment (jdhughes-usgs) [inline doc/mf6io/mf6ivar/dfn/gwf-uzf.dfn:65]: Why not spell out `water_content` instead of cryptic `wc`?

File: doc/ReleaseNotes/ReleaseNotes.tex [hunk 2]
```diff
@@ -159,15 +159,15 @@ \section{History}
...
+	        \item A new option for printing water contents to a dedicated output file has been added to UZF.  To activate, the keyword WATER_CONTENT is added to the OPTIONS block of UZF, followed by FILEOUT, followed by the user-specified output file name, for example 'water-content.uzf.bin'.  The approach is analogous to the STAGE option within the SFR options block.  Contents of the new file will be written in binary and can be read using flopy's binaryfile utility.  A new test, located in the autotest directory and titled est_gwf_uzf_wc_output.py demonstrates how to read the new binary water content output file.
```
Comment (langevin-usgs) [inline doc/ReleaseNotes/ReleaseNotes.tex:162]: These don't look like valid quotes for latex.  You need to use `` and '' I believe.

File: doc/ReleaseNotes/ReleaseNotes.tex [hunk 3]
```diff
@@ -159,14 +159,15 @@ \section{History}
...
 	        \item The residual balance error for groundwater flow and solute transport is now written to the diagonal position of the flowja array.  The flowja array is optionally written to the binary model budget file according to user settings in the output control file and other package input files.
 	\end{itemize}
 	
 	\underline{EXAMPLES}
 	\begin{itemize}
 	        \item Added the following new examples: 
 	        \begin{itemize}
-	          \item --
+	          \item -- A new test compares MODFLOW~6 water contents saved with the new WATER_CONTENT keyword in the OPTIONS block of UZF to water contents calculated by MF-NWT and written to the linker file (MT3D-USGS) by the LMT package.  The name of the new test is test_gwf_uzf_wc_output.py
```
Comment (langevin-usgs) [inline doc/ReleaseNotes/ReleaseNotes.tex:170]: This section is for new examples (from the modflow6-examples repo), not new autotests.

File: doc/mf6io/gwf/binaryoutput.tex [hunk 4]
```diff
@@ -275,6 +275,7 @@ \subsubsection{Advanced Flow and Transport Packages}
 GWF/LAK & STAGE & Simulated lake stage  \\
 GWF/SFR & STAGE & Simulated stream reach stage  \\
 GWF/MAW & HEAD & Simulated well head  \\
+GWF/UZF & WATER CONTENT & Simulated unsaturated zone cell water content \\
```
Comment (langevin-usgs) [inline doc/mf6io/gwf/binaryoutput.tex:278]: Isn't the header supposed to be WATER-CONTENT. This should be the text string written to the binary file.

File: doc/ReleaseNotes/ReleaseNotes.tex (hunk 3 above)
Comment (emorway-usgs) [reply doc/ReleaseNotes/ReleaseNotes.tex]: I have moved some of this material up to the NEW FUNCTIONALITY section.  Fixed with f0b161b.  I now understand this section to be where the Hecht-Mendez problem would need to be listed...

File: doc/mf6io/gwf/binaryoutput.tex (hunk 4 above)
Comment (emorway-usgs) [reply doc/mf6io/gwf/binaryoutput.tex]: Great catch, fixed with f0b161b

File: doc/mf6io/mf6ivar/dfn/gwf-uzf.dfn (hunk 1 above)
Comment (emorway-usgs) [reply doc/mf6io/mf6ivar/dfn/gwf-uzf.dfn]: addressed by 03b36d1

**Pull request merged**